### PR TITLE
Navi app style update

### PR DESCRIPTION
**Pull request by mattgraham1995**
## Description
Style changes in navi added a gray background to dashboards

## Proposed Changes

- Adjust padding so that gray background stretches to left and right edges of window

## Screenshots
Screen Shot 2019-06-17 at 11 47 58 AM
Screen Shot 2019-06-17 at 11 48 03 AM
Screen Shot 2019-06-17 at 11 48 14 AM
Screen Shot 2019-06-17 at 11 48 09 AM

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/app/app/styles/views/reports.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+.navi-report {
+  &__breadcrumb,
+  &__header,
+  &__footer {
+    padding: 0 20px;
```
Comment (balajimanoharan) [inline packages/app/app/styles/views/reports.less]: Can we make this the same as the dashboard?
https://github.com/yahoo/navi/pull/410/files#diff-035417c76de71435bce60d8f3cb07391R12

File: packages/app/app/styles/views/reports.less [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  }
+
+  .report-builder {
+    padding: 5px 15px;
```
Comment (balajimanoharan) [inline packages/app/app/styles/views/reports.less]: And should this also be the same as the dashboard body?

File: packages/app/app/styles/views/reports.less (hunk 1 above)
Comment (mattgraham1995) [reply packages/app/app/styles/views/reports.less]: I intended to preserve the original padding while extending the gray background, but yeah I can make them more consistent

**Comment by jkusa** [comment]
Can we add equal padding around the report view?

**Comment by mattgraham1995** [comment]
@jkusa You mean like this?
Screen Shot 2019-06-26 at 11 13 49 AM

**Comment by jkusa** [comment]
Is the dashboard index view up to date? Looks like the title needs to move to the left a little

**Comment by balajimanoharan** [comment]
Can we add the gray backgrounds to the index views as well? It would be nice if it was consistent.

**Pull request merged**